### ISSUE TITLE
Can't modify an image slide without reuploading the image

### ISSUE THREAD

**Issue by aLxSpir** [PrestaShop/PrestaShop#34660]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [x] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Context is Multi-shop

When modifying an already existing image slide, I can't modify the description only, I also have to re-select an image file. Otherwise, an exception is thrown : "Path cannot be empty".

![Capture d'écran 2023-11-23 130021](https://github.com/PrestaShop/PrestaShop/assets/47948398/3f77fbbe-916d-420a-a202-e605f24c71b8)

### Expected behavior

I should be able to modify the image slide without having to reupload the image. 

### Steps to reproduce

1. Go to configuration in ps_imageslider
2. Click on Modify on an existing image slide
3. Click on "Save"

### PrestaShop version(s) where the bug happened

8.1.2

### PHP version(s) where the bug happened

8.1.18

### If your bug is related to a module, specify its name and its version

ps_imageslider v3.1.3 (latest)

**Comment by SharakPL** [comment]
> ### PrestaShop version(s) where the bug happened
> 
> 1.8.2

Do you mean 8.1.2?

**Comment by aLxSpir** [comment]
Yes I do ! 8.1.2 :)

**Comment by SharakPL** [comment]
Did you add new languages after creating these slides? Each language requires an image and they aren't automatically duplicated on new languages so you need to reupload an image for each new language. After that simple change of description should be possible without problems.

**Comment by aLxSpir** [comment]
I didn't add any new language. Also, I get the error every time unless I re-select an image.

**Comment by SharakPL** [comment]
Can you provide a screen record for better understanding? I tested this module on multistore multilanguage configuration (develop branch) and it all works just fine. Even when specific language doesn't have an image set you still can update only the description.

Don't you have some modifications in your module? Error references line 462 in ps_imageslider.php but there's nothing there in v.3.1.3:

![obraz](https://github.com/PrestaShop/PrestaShop/assets/5007456/609b8e0c-ed34-4a65-9135-38c27ebac4bf)

**Comment by aLxSpir** [comment]
Indeed, I'm sorry for having wasted your time... It seems the version of the module was actually 3.0.0 but was displayed as 3.1.3 in the admin panel. I have no clue how it happened since it was a fresh install of Prestashop 8.1.2 ...

By the way, in multi-store context, when I duplicate an existing store, image slides can be duplicated as well and are then shown as "shared image slides", but it seems there is no possibility to unshare them or extend the sharing to other existing stores. Am I missing something ?

**Comment by SharakPL** [comment]
> By the way, in multi-store context, when I duplicate an existing store, image slides can be duplicated as well and are then shown as "shared image slides", but it seems there is no possibility to unshare them or extend the sharing to other existing stores. Am I missing something ?

No, that's exactly what I've experienced. I may have described the behaviour from an older versions in one of my previous posts.

How did you install the shop? It's impossible to have v.3.0.0 of ps_imageslider installed with it. This version was released back in 2018. Looks like the module's files were overriden manually. It was v.3.1.3 on the Modules list because that's the version installed originally on the shop. Version numbers are stored in the database and are updated when module is installed/updated properly, not manually overriden.
